Index: all/modules/_fields/soundcloudfield/soundcloudfield.module
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- all/modules/_fields/soundcloudfield/soundcloudfield.module	(revision bf2e5dd8ed9ce2219af1692980d588abe0986c29)
+++ all/modules/_fields/soundcloudfield/soundcloudfield.module	(date 1568022500003)
@@ -416,7 +416,7 @@
       break;
     // HTML5 (classic+visual) player case.
     case 'soundcloud_html5':
-      $oembed_endpoint = 'http://soundcloud.com/oembed';
+      $oembed_endpoint = 'https://soundcloud.com/oembed';
 
       // Get HTML5 player-specific settings.
       $html5_player_height = (empty($settings['html5_player']['html5_player_height']) ? SOUNDCLOUDFIELD_DEFAULT_HTML5_PLAYER_HEIGHT : $settings['html5_player']['html5_player_height']);
